### DISCUSSION TITLE
Assigning computed feature values to all rows belonging to a group

### DISCUSSION THREAD

**Discussion by kar10tik** [blue-yonder/tsfresh#1076] [Q&A]
Tsfresh provides a lot of useful features from an input DataFrame (df). But I'm looking for something similar to `pd.df.groupby.transform`, which allows me to compute features for each group and helps assign, in a new column, the value of the computed feature to all rows belonging to that group:

`groups = df.groupby(group_cols)`
`df["Std"] = groups["column"].transform('std') # Standard deviation as example`

Pandas provides this groupby transform feature for a few statistical features. I wish to use tsfresh to compute comprehensive features and assign the feature values to each row belonging to the group in a new column. How can I do this?

**Comment by nils-braun** [comment]
You might be able to do this with pandas-own operations. I think using `pd.groupby.apply` with a `extract_features` call might be helpful here.
```python
tmp = df.groupby(...).apply(lambda x: extract_features(x, ...).droplevel(0)
```
After that, the `tmp` dataframe has the "id" column as index, so you can just join it with the original data. I think this should give you what you want (?)
